### PR TITLE
Scrollable cell for firefox

### PR DESCRIPTION
**Pull request by v-hclaveria**
Proposal to workaround issue with scroll bar on td element on Firefox. This seems to be a known issue, more info: https://bugzilla.mozilla.org/show_bug.cgi?id=221154

### CODE REVIEW

**Comment by claud-io** [comment]
@v-hclaveria check the conflict, also, that solution is going to always show the scrollbar, we need it only when space is not enough

File: www/resources/views/pages/testhistory.blade.php [hunk 1]
```diff
@@ -52,14 +52,16 @@
                             <th><input type="checkbox" name="t[]" value="{{ $record->getTestId() }}" aria-label="Select this test" /></th>
                             <td class="url"><a href="/result/{{ $record->getTestId() }}/">{{ $record->getUrl() }}</a></td>
                             <td class="date">{{ date_format(date_create($record->getStartTime()), 'M d, Y g:i:s A e') }}</td>
-                            <td class="location">{{ $record->getLocation() }}</td>
+                            <td class="location"><div id="scrollable-wide-wrap">
```
Comment (claud-io) [inline www/resources/views/pages/testhistory.blade.php:55]: not sure why we need two div, also the scrollbar that way is overlapping with cell content, below solution works better please check
```
<td class="location" style="padding: unset;">
       <div style="overflow-x: auto;padding: 1em;">Test Scheduler 1182 - Chrome - 3GFast</div>
</td>
```

**Comment by v-hclaveria** [comment]
image

File: www/assets/css/pagestyle2.css [hunk 2]
```diff
@@ -6643,6 +6643,14 @@ div.overflow-container > *,
   position: relative;
 }
 
+.scrollable-location {
+    padding: unset;
+}
+.scrollable-location-content {
```
Comment (claud-io) [inline www/assets/css/pagestyle2.css:6649]: add space

File: www/resources/views/pages/testhistory.blade.php [hunk 3]
```diff
@@ -52,14 +52,16 @@
                             <th><input type="checkbox" name="t[]" value="{{ $record->getTestId() }}" aria-label="Select this test" /></th>
                             <td class="url"><a href="/result/{{ $record->getTestId() }}/">{{ $record->getUrl() }}</a></td>
                             <td class="date">{{ date_format(date_create($record->getStartTime()), 'M d, Y g:i:s A e') }}</td>
-                            <td class="location">{{ $record->getLocation() }}</td>
+                            <td class="location scrollable-location">
```
Comment (claud-io) [inline www/resources/views/pages/testhistory.blade.php:55]: format

**Review by claud-io — APPROVED** [review]

**Pull request merged**